README.md

```diff
@@ -120,6 +126,9 @@ git submodule update --init --recursive
 set PATH=<protobuf_install_dir>\bin;%PATH%
 set USE_MSVC_STATIC_RUNTIME=0
 
+# use the static installed protobuf
+set CMAKE_ARGS=-DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DProtobuf_USE_STATIC_LIBS=ON
```

Comment:
Defaults for `-DProtobuf_USE_STATIC_LIBS` and `-DONNX_USE_PROTOBUF_SHARED_LIBS` are not working... Thus the setting here is required.